src/io-util.cc

```diff
@@ -153,9 +154,10 @@ bool MMapFile(const std::string &filepath, MMapFileHandle *handle,
 
 #if TINYUSDZ_MMAP_SUPPORTED
 #if defined(_WIN32)
-  // int fd = open(filepath.c_str(), writable ? O_RDWR : O_RDONLY);
+  std::basic_string<TCHAR> tFilepath(filepath.begin(), filepath.end());  // Using TCHAR string to automatically use normal or wide characters if UNICODE is enabled
+
   HANDLE hFile =
-      CreateFile(filepath.c_str(), GENERIC_READ, FILE_SHARE_READ, nullptr,
+      CreateFile(tFilepath.c_str(), GENERIC_READ, FILE_SHARE_READ, nullptr,
```

Comment:
Instead of  possibly converting the filename to a char or wide character, why no use [CreateFileA](https://learn.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-createfilea) directly so it always uses `char`?